src/assetsdir.cpp

```diff
@@ -37,7 +37,7 @@ void CAssetsDir::SetHex(const std::string& assetHex, const std::string& label)
     Set(CAsset(uint256S(assetHex)), AssetMetadata(label));
 }
 
-void CAssetsDir::InitFromStrings(const std::vector<std::string>& assetsToInit)
+void CAssetsDir::InitFromStrings(const std::vector<std::string>& assetsToInit, const std::string& peggedAsset = "bitcoin")
```

Comment:
I don't think we need to default to "bitcoin" here?

Reply:
agreed, will remove.